### PR TITLE
[#665] Build UI aspect of channels page

### LINKED ISSUE #665: UI aspects of Channels page within Admin
### Requirements

Before channels are able to be connected, we need to list all of the possible channels on the Channels page within Admin. These channels will all be visibly listed on the lefthand side of the page. In this view, we need to offer the option of a button to add more accounts to each channel.
1 0 0 Connect channel@2x

After channels are successfully connected, we need to create a list of the connected Airy Live Chat/Facebook Messenger/SMS/WhatsApp/Google Business Messages accounts that are currently connected to Core. We should also display how many accounts are connected from each channel. In this view, we need to still offer the button to add more accounts to each channel.
7 0 0 Connected channels@2x

For design purposes, buttons and filters will not be highlighted unless the cursor hovers on them. Once the cursor hovers over the options such as:
- any listed channel source
- Add a channel
- Filter
- Delete
- Search

then these items will become highlighted to display that they are being utilized.
Z Interactions@2x

### PR DESCRIPTION
**Pull request by ogbeche77**
Fixes #665

**`All channels are first listed when users first visit the Channels page`**

screen shot1

**`This is when users have successfully connect to various channel sources, they also have the possibility to add more accounts`**

Screen shot2

### CODE REVIEW

File: frontend/ui/src/pages/Channels/ChannelsSources/ChannelsConnected.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          <div className={styles.connectedContainer}>
+            <div className={styles.connectedSum}>
+              <p>
+                {props.showSumOfChannels} {props.connected}
+              </p>
+            </div>
+            <div className={styles.connectedChannelBox}>
+              <div className={styles.connectedChannel}>
+                {props.connectedChannel.map((channel: Channel) => {
+                  const channelName = channel.metadata.name;
```
Comment (AitorAlgorta) [inline frontend/ui/src/pages/Channels/ChannelsSources/ChannelsConnected.tsx:34]: maybe just use `channel.metadata.name`?

File: frontend/ui/src/pages/Channels/ChannelsSources/ChannelsConnected.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                          {!props.ignoreChannelId && <div className={styles.channelId}>{channel.sourceChannelId}</div>}
+                        </div>
+                      </li>
+                    </>
+                  );
+                })}
+              </div>
+              <div className={styles.extraChannelButton}>
+                {props.extraChannel && (
+                  <button className={styles.extraChannel} title="Click for more details">
```
Comment (AitorAlgorta) [inline frontend/ui/src/pages/Channels/ChannelsSources/ChannelsConnected.tsx:58]: why create a new button? we already have this one in our library https://components.airy.co/?path=/story/airy-components-cta-linkbutton--link-button-component

File: frontend/ui/src/pages/Channels/ChannelsSources/ChatPluginSource.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      <ChannelDetails
+        title="Airy Live Chat "
+        text="Best of class browser messenger"
+        image={<AiryLogo />}
+        buttonIcon={<AddChannel />}
+        displayButton={chatPluginSources.length === 0}
+      />
+
+      <ChannelsConnected
+        showConnectedChannels={chatPluginSources.length > 0}
```
Comment (AitorAlgorta) [inline frontend/ui/src/pages/Channels/ChannelsSources/ChatPluginSource.tsx:27]: I think you could reduce the number of props a lot if you just make some small changes

**Review by AitorAlgorta — CHANGES_REQUESTED** [review]

**Review by AitorAlgorta — APPROVED** [review]
LGTM 🚀

**Review by bitboxer — APPROVED** [review]
👍 🥳 Awesome.

**Pull request merged**